homeworks/src/homework11/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,49 @@
+package homework11.coffee.order;
+
+import java.util.Comparator;
+import java.util.PriorityQueue;
+import java.util.Queue;
+
+public class CoffeeOrderBoard {
+    private Queue<Order> orderBoard;
+    private static int counter = 1;
```

Comment:
`static` there has sense if you should have unic numeration for multiple coffees shops
If this class will be used for multiple not related entities counter should be non-static